src/main/features/claimant-response/helpers/settlementHelper.ts

```diff
@@ -44,18 +44,18 @@ export function prepareDefendantPartyStatement (claim: Claim, draft: DraftClaima
 export function prepareDefendantOffer (claim: Claim, draft: DraftClaimantResponse): Offer {
   const response: FullAdmissionResponse | PartialAdmissionResponse = claim.response as FullAdmissionResponse | PartialAdmissionResponse
 
+  const amount = response.responseType === ResponseType.PART_ADMISSION ? NumberFormatter.formatMoney(response.amount) : NumberFormatter.formatMoney(claim.totalAmountTillToday)
   if (response.paymentIntention.paymentDate) {
     const completionDate: Moment = response.paymentIntention.paymentDate
-    const amount = response.responseType === ResponseType.PART_ADMISSION ? NumberFormatter.formatMoney(response.amount) : 'the full amount'
     const content: string = `${response.defendant.name} will pay ${amount}, no later than ${MomentFormatter.formatLongDate(completionDate)}`
     return new Offer(content, completionDate, response.paymentIntention)
   } else if (response.paymentIntention.repaymentPlan) {
     const paymentPlan: PaymentPlan = PaymentPlanHelper.createPaymentPlanFromClaim(claim, draft)
     const instalmentAmount: string = NumberFormatter.formatMoney(paymentPlan.instalmentAmount)
-    const paymentSchedule: string = PaymentScheduleTypeViewFilter.render(response.paymentIntention.repaymentPlan.paymentSchedule)
+    const paymentSchedule: string = PaymentScheduleTypeViewFilter.render(response.paymentIntention.repaymentPlan.paymentSchedule).toLowerCase()
     const firstPaymentDate: string = MomentFormatter.formatLongDate(paymentPlan.startDate)
     const completionDate: Moment = paymentPlan.calculateLastPaymentDate()
-    const content: string = `${response.defendant.name} will pay instalments of ${instalmentAmount} ${paymentSchedule}. The first instalment will be paid by ${firstPaymentDate}.`
+    const content: string = `${response.defendant.name} will repay ${amount} in instalments of ${instalmentAmount} ${paymentSchedule}. The first instalment will be paid by ${firstPaymentDate}.`
```

Comment:
we usually do content/text in *.njk, i am not sure why its done in .ts in first place. Could you please check if this can be moved to *.njk? @sabahirfan @dharmendrak what do you guys think?

Reply:
AFAIK the content of the offer is created in the ts file. in NJK file, they are just showing  the offer content. So it makes sense to me. If you guys want to change, we have to change it everywhere.

Reply:
@karthikkannankainos please put this content in .njk file, so that we can put the string in t() block, so that we can add translation for it

Reply:
Discussed with @dharmendrak and will do the change as separate PR as its unrelated to this change.

Reply:
https://tools.hmcts.net/jira/browse/ROC-4688 created for the above